scripts/bootstrap.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh
+
+# Script to install Nix/Devbox/Direnv
+#
+#
+export DEVBOX_USE_VERSION="0.7.1"
```

Comment:
Why are we forcing a specific version of devbox? Wouldn't it make more sense to always install latest?

Reply:
I would like our customers (devs) have guarantee working experience rather than randomly broken script that needs to figure out what changes in Devbox causing them issue.

BTW my intent for this script is they use a curl statement to call it so we can update the devbox version when we think it's stable enough

Reply:
So far devbox has been stable enough and I suspect we don't have the time as a team to follow up and check each devbox release is stable. I'd rather just tell everyone to stay on the latest version unless there's a known issue